portlets/src/main/frontend/src/apps/profileStats/components/SpaceCommonsItems.vue

```diff
@@ -0,0 +1,120 @@
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-row user-item>
+    <template>
+      <v-list-item
+        :id="id"
+        :key="commonSpace.id"
+        href="url"
+        class="py-0 px-2">
+        <v-list-item-avatar
+          class="my-1 mr-2"
+          size="30">
+          <v-img :src="!skeleton && avatarUrl || ''" :class="skeleton && 'skeleton-background'" />
+        </v-list-item-avatar>
+
+        <v-list-item-content class="py-0" :href="url">
+          <v-list-item-title>
+            <a
+              :class="skeleton && 'skeleton-background skeleton-text skeleton-list-item-title skeleton-border-radius'"
+              class="font-weight-bold subtitle-2 request-user-name darken-2"
+              :href="url">
+              {{ commonSpace.displayName }}
+            </a>
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </template>
+  </v-row>
+</template>
+
+<script>
+  const randomMax = 10000;
+  export default {
+    props: {
+     commonSpace: {
+       type: Object,
+       default: () => null,
+     },
+     skeleton: {
+       type: Boolean,
+       default: false,
+     },
+    },
+    data() {
+     return {
+      id: `spaceItem${parseInt(Math.random() * randomMax)
+        .toString()
+        .toString()}`,
+     }
+    },
+    computed: {
+    url() {
+      if (!this.commonSpace || !this.commonSpace.groupId) {
+        return '#';
+      }
+      const uri = this.commonSpace.groupId.replace(/\//g, ':');
+      return `${eXo.env.portal.context}/g/${uri}/`;
+    },
+    avatarUrl() {
+      return this.commonSpace && this.commonSpace.avatarUrl || `${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spaces/${this.commonSpace.avatarUrl}/avatar`;
+    },
+    labels() {
```

Comment:
Could you make labels as a property instead of computed like in https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/common/components/ExoSpaceAvatar.vue#L56

Reply:
done!